infrastructure/.dockerignore

```diff
@@ -1 +0,0 @@
-data
```

Comment:
I removed entire infrastructure folder - since now we have admin-bro-dev it is not needed